nursery/hash-data-using-fnv.yml

```diff
@@ -14,9 +14,9 @@ rule:
         - number: 16777619 = 32 bit FNV_prime
         - number: 1099511628211 = 64 bit FNV_prime
         # FNV-0 hash does not use these initialization values
-        - optional:
-          - number: 2166136261 = 32 bit offset_basis
-          - number: 14695981039346656037 = 64 bit offset_basis
+      - optional:
```

Comment:
wrong indentation; per the documentation: "`optional` is an alias for `0 or more`"